style/cell.less

```diff
@@ -1,5 +1,5 @@
 .rdg-cell {
-  display: inline-block;
+  contain: strict;
```

Comment:
We don't need to do the checking like this? https://github.com/adazzle/react-data-grid/pull/2014/files#diff-95d4fa4e583fb3fa7c9d33f79b431c52R23

Reply:
I've removed `display: inline-block` as it's redundant with `display: flex` on the rows.
It shouldn't break anything if `contain; strict` isn't supported here.

Reply:
Cool. I'm still not sure what does the `contain: strict` really do here. I'm still reading the doc that you linked. Based on your screenshots, this looks nice.